scripts/zones/Mhaura/npcs/Numi_Adaligo.lua

```diff
@@ -1,7 +1,6 @@
 -----------------------------------
 -- Area: Mhaura
 --  NPC: Numi Adaligo
---  Involved In Quest: RYCHARDE_THE_CHEF
```

Comment:
I'd say to probably leave these comments here, otherwise it isn't clear what the NPC does unless you google it or search the whole codebase. I don't know if we've decided we're going to delete "empty husk" NPCs if all their involved quests are written

Reply:
Mmm, I'm thinking about this... what if I change the coment to the quest file location instead?
so:
-- Involved IN Quest: RYCHARDE_THE_CHEF
Would change to:
-- Used in: scripts/quests/otherAreas/Rycharde_the_Chef.lua

Reply:
Yup, sounds good to me 👍